metadata/edgexfoundry.md

```diff
@@ -10,34 +10,30 @@ This snap is a part of EdgeX Foundry and contains the following reference servic
 * Core Data
 * Core Command
 * Core Metadata
-* Registry and Config (Consul)
+* Registry and Config Provider (Consul)
 * Message Bus (Redis)
 
 **Supporting Services**
-* Rules Engine (eKuiper)
+* Rules Engine (eKuiper) - Deprecated in favor of https://snapcraft.io/edgex-ekuiper
 * Scheduling
 * Alerts & Notifications
 
 **Application Services**
-* Configurable Application Service
+* Configurable Application Service (for Rules Engine event filtering) - Deprecated in favor of https://snapcraft.io/edgex-app-service-configurable
```

Comment:
ASC is not only for rules engine event filtering but also for http-export, mqtt-export etc.
https://github.com/edgexfoundry/app-service-configurable/tree/main/res

Reply:
The one embedded in edgexfoundry app is for rules engine only. The hooks make sure of that.